pkg/validation/port_values.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package validation
+
+import (
+	"fmt"
+
+	"github.com/projectriff/cli/pkg/cli"
+)
+
+func PortValues(port int32, field string) cli.FieldErrors {
```

Comment:
The method only accepts a single value, so it should not be plural. The "value" is redundant, what else would it validate?

```suggestion
func Port(port int32, field string) cli.FieldErrors {
```

or

```suggestion
func PortNumber(port int32, field string) cli.FieldErrors {
```

Reply:
Should we rename the file as well to just `pkg/validation/port.go`